content/articles/the-complete-roadmap-to-blockchain-development/index.md

```diff
@@ -0,0 +1,116 @@
+
+ The good thing about smart contracts is that you can move money within itself. To better understand this, let us say that you had normal code which required computational power for which the users would need to pay a small amount. You would first need to integrate it with a payment service such as Skrill, Neteller, or Paypal but first, you would need permission for that. You would also be constrained by the API in this scenario. However, in a smart contract, you can write any logic for moving the money around and distributing the fee charged to the owners who have dedicated their computers to do the complex computations in changing the data of the smart contract.
+
+**Solidity**
+
+ Knowing how to write smart contracts is the most important thing in blockchain app development. The demand for smart contracts in several fields, not just finance has led to an increased salary for developers hitting the highs of $250,000 annually. There are a couple of programming languages used in writing a smart contract but the most popular one is `solidity`. The syntax is similar to that of JavaScript which could be quite misleading since it works very differently from Javascript. Solidity is also limited as compared to Javascript thus you should avoid doing very complex things.
+
+ For example, this is how you write `Hello World` in solidity
+ ```js
+ // My First Smart Contract 
+pragma solidity >=0.5.0 <0.7.0;
+contract HelloWorld {
+    function get()public pure returns (string memory){
+        return 'Hello Contracts';
+    }
+}
+ ```
+ I hope you can note the similarities to Javascript i.e the functions, the methods, the curly braces, etc. 
+
+ A great way to experiment with solidity is to use an online IDE called Remix. With Remix You just load the [website](https://remix.ethereum.org/), start coding, and run your first smart contract. [Here](https://www.tutorialspoint.com/solidity/index.htm) is a reference in case you want to get started with programming in solidity. However, in real-life projects, we need something more robust and thus most developers use Truffle which is a popular framework for smart contracts. "Truffle" is a command-line tool written in NodeJs and it can easily be installed on Windows, Mac OS, and Linux using the package manager npm.
```

Comment:
Node.js.......in another paragraph you have written NPM, but here you have written npm. Use one of them consistently in the entire article.